postgres_logger.go

```diff
@@ -0,0 +1,80 @@
+package tracelog
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+)
+
+type PostgresLogger struct {
```

Comment:
Why we need separate struct for pg logger? Can we just provide the correct writer to the constructor of the main one?